Off-Top-App/off-top-python
# UPDATE-MASTER | meshed flaskServer.py and app.py into one branch

The purpose of this branch is to update master and no longer use myFlask

**How to test**:
1. run `python app.py`
2. Enter `http://localhost:5000/sparkpi` to assure spark streaming still works
3. Enter `http://localhost:5000/sessions` to assure that mongodb instance is still connected as you should see json objects appear in the browser
